tests/test_asdf_parameters.py

```diff
@@ -2,11 +2,11 @@
 from pathlib import Path
 import pytest
 
-from jwst.stpipe.config_parser import ValidationError
-from jwst.stpipe.step import Step
+from stpipe.config_parser import ValidationError
+from stpipe.step import Step
 
-from .steps import MakeListStep, MakeListPipeline
-from .util import t_path
+from steps import MakeListStep, MakeListPipeline
+from util import t_path
```

Comment:
Shouldn't these remain relative imports as these modules are in the `tests` dir?

Reply:
The tests directory isn't a Python package, so relative imports don't work here.  Python isn't able to resolve the `.` into anything.

Reply:
Ah ok!